doc/docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -61,27 +63,27 @@ neo4j.authentication.basic.username neo4j
 
 
 
-== Neo4j Driver Options
+== Neo4j driver options
 
-Under the covers, the spark connector uses the link:https://neo4j.com/docs/driver-manual/current/get-started/#driver-get-started-about[official Neo4j Java Driver].  As such, in many situations you'll want the control to set driver options to account for your production deployment of Neo4j, and how to communicate with it.   This is done using the `options` example above.
+Under the covers, the Spark connector uses the link:https://neo4j.com/docs/java-manual/current/get-started/#java-driver-get-started-about[official Neo4j Java Driver].  In many situations you want the control to set driver options to account for your production deployment of Neo4j, and how to communicate with it. This can be done using the `options` example above.
```

Comment:
Please, take a look at this link. I've changed it because if you clicked the previous one you'd be redirected to the docs page, not to any specific driver manual.